examples/validate_relative_binding.py

```diff
@@ -471,9 +329,9 @@ def predict_dG(results: dict):
                     assert 0, "Unknown property units"
             except Exception as e:
                 print(f"Unable to find property {cmd_args.property_field}: {e}")
-        print(f"Epoch: {epoch}, Processing Mol: {mol_name}, Label: {label_dG}")
+        print(f"Epoch: {epoch}, Processing Mol: {mol_name}, Label: {label_dG}, Seed {rabfe_future.seed}")
```

Comment:
We now log the seed that the simulation was run with, allowing for reproducing the simulations